projects/organization-management/src/app/store/users/users.effects.spec.ts

```diff
@@ -33,13 +52,20 @@ describe('Users Effects', () => {
     usersService = mock(UsersService);
     when(usersService.getUsers()).thenReturn(of(users));
     when(usersService.getUser(anything())).thenReturn(of(users[0]));
+    when(usersService.addUser(anything())).thenReturn(of(users[0]));
+    when(usersService.updateUser(anything())).thenReturn(of(users[0]));
 
     TestBed.configureTestingModule({
       declarations: [DummyComponent],
       imports: [
         CoreStoreModule.forTesting(['router']),
+        CustomerStoreModule.forTesting('user'),
         OrganizationManagementStoreModule.forTesting('users'),
-        RouterTestingModule.withRoutes([{ path: 'users/:B2BCustomerLogin', component: DummyComponent }]),
+        RouterTestingModule.withRoutes([
+          { path: 'users/:B2BCustomerLogin', component: DummyComponent },
+          { path: 'account/organization/users', component: DummyComponent },
+          { path: 'account/organization/users/1', component: DummyComponent },
```

Comment:
these are required for the redirecting after edit/create, right? I'll have a look into that.